src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -39,13 +41,23 @@ protected function configure()
                 new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
                 new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of places)'),
                 new InputOption('label', 'l', InputArgument::OPTIONAL, 'Labels a graph'),
+                new InputOption('format', null, InputArgument::OPTIONAL, 'The dump format', 'dot'),
+                new InputOption(
+                    'transition',
+                    null,
+                    InputArgument::OPTIONAL,
+                    'The transition format for PUML format (arrow|square)',
+                    PlantUmlDumper::DEFAULT_TRANSITION_TYPE
+                ),
             ))
             ->setDescription('Dump a workflow')
             ->setHelp(<<<'EOF'
 The <info>%command.name%</info> command dumps the graphical representation of a
-workflow in DOT format
+workflow in different formats
 
-    %command.full_name% <workflow name> | dot -Tpng > workflow.png
+<info>DOT</info>:  %command.full_name% <workflow name> | dot -Tpng > workflow.png 
+<info>PUML</info>: %command.full_name% <workflow name> --format=puml -p | java -jar plantuml.jar > workflow.png
```

Comment:
What is `-p` ?

Reply:
That is a typo (not present in the PR doc) the `-p`, as mentioned in the PR doc is for `piping`. But it is for plantuml.jar... not for the command. I have fixed that too.